pullrequest.rst

```diff
@@ -166,6 +166,11 @@ any discussion of what the pull request is trying to solve (e.g. fixing a
 spelling mistake), then the pull request needs to have the "trivial" label
 added to it.
 
+If your pull request involves several commits as a result of addressing code
+review comments, please do not squash them.  When you squash your commits,
```

Comment:
```git commit --amend; git push --force``` also does a squash (and may be the more common reason that people squash). I'd prefer that we use a positive form of wording, something like "please keep the full history of your PR intact (i.e. don't "squash" or amend the commits, but keep them separate)"